R/geom.R

```diff
@@ -245,6 +245,10 @@ st_relate	= function(x, y, pattern = NA_character_, sparse = !is.na(pattern)) {
 #' If \code{y} is missing, `st_predicate(x, x)` is effectively called, and a square matrix is returned with diagonal elements `st_predicate(x[i], x[i])`.
 #'
 #' Sparse geometry binary predicate (\code{\link{sgbp}}) lists have the following attributes: \code{region.id} with the \code{row.names} of \code{x} (if any, else \code{1:n}), \code{ncol} with the number of features in \code{y}, and \code{predicate} with the name of the predicate used.
+#'
+#' @note For intersection on pairs of simple feature geometry sets, use
```

Comment:
I would say "geometries" rather than "geometry sets", since we don't do the intersection of the _set_s, but of the pairs of geometries.

Reply:
OK, I'm changing it